wagtail/documents/templates/wagtaildocs/multiple/add.html

```diff
@@ -10,7 +10,7 @@
 {% endblock %}
 
 {% block main_content %}
-    <div class="drop-zone w-mt-8">
+    <div class="drop-zone w-mt-8" data-controller="w-zone" data-w-zone-active-class="hovered" data-action="dragover->w-zone#activate:prevent dragleave->w-zone#deactivate dragend->w-zone#deactivate drop->w-zone#deactivate:prevent drop@document->w-zone#noop:prevent dragover@document->w-zone#noop:prevent">
```

Comment:
Just a subtle note here, you should not need to add `prevent` twice for the same event, adding `prevent` on the `@document` ones is sufficient.

Let's also put the `@document` listeners first, these will be attached first and allows the logic to be a bit more predictable.

So.. the document events are `noop` and are just there to prevent the default of these events, but that will not stop the propagation.

The local (non`@document`) events are there to control the mode changes, and only listen for events from within the container.

```
        data-action="drop@document->w-zone#noop:prevent dragover@document->w-zone#noop:prevent drop->w-zone#deactivate dragover->w-zone#activate dragleave->w-zone#deactivate dragend->w-zone#deactivate"
```